src/components/BasicList.tsx

```diff
@@ -25,24 +26,25 @@ const BasicList: React.FC<Props> = ({
   onPress,
   onPressTrashIcon,
   patientList,
+  technologyList,
 }) => {
   return (
     <View style={styles.contanier}>
-      {data?.length || patientList?.length ? (
+      {data?.length || patientList?.length || technologyList?.length ? (
```

Comment:
```suggestion
      {data?.length || patientList?.length || technologyList?.length ? (
```
Essa linha vai mudar um pouco no PR do Felipe, ajustamos lá, na integração.